src/main/java/at/hannibal2/skyhanni/features/garden/visitor/VisitorListener.kt

```diff
@@ -171,7 +188,32 @@ class VisitorListener {
         if (!GardenAPI.onBarnPlot) return
         if (!VisitorAPI.inInventory) return
         val visitor = VisitorAPI.getVisitor(lastClickedNpc) ?: return
+        val isRefuseSlot = event.itemStack.toString() == "1xtile.clayHardenedStained@14"
+        val isAcceptSlot = event.itemStack.toString() == "1xtile.clayHardenedStained@13"
```

Comment:
this weirdness can prob be avoided by using `LorenzToolTipEvent`

Reply:
What's wrong with this?

Reply:
What is correct in this?